orm/schemas/globals-1.json

```diff
@@ -10,6 +10,12 @@
           "type": "string",
           "format": "hostname"
         },
+        "max_server_timeout": {
+          "type": "integer"
+        },
+        "timeout_server_default": {
+          "type": "integer"
+        },
```

Comment:
I suggest naming the variables consistently. Either:
```
timeout_server_max
timeout_server_default
```
or
```
server_timeout_max
server_timeout_default
```

Reply:
I agree. I like the first one better. Since it maps directly to timeout_server in haproxy any forthcoming timeouts can keep follow that naming without issues. The second option might be abit more open for some confusion. But either works in my mind as long as it is well documented.

Reply:
I agree as well about the variable naming - no otherwise looks great! Nice example in the docs as well.

Reply:
resolved in #40